controllers/activemqartemis_controller_test.go

```diff
@@ -121,6 +122,84 @@ var _ = Describe("artemis controller", func() {
 		}
 	})
 
+	Context("pod disruption budget", Label("pod-disruption-budget"), func() {
+		It("pod disruption apply number", func() {
+			minOne := intstr.FromInt(1)
+			pdb := policyv1.PodDisruptionBudgetSpec{
+				MinAvailable: &minOne,
+			}
+
+			brokerCr, createdCr := DeployCustomBroker(defaultNamespace, func(candidate *brokerv1beta1.ActiveMQArtemis) {
+				candidate.Spec.DeploymentPlan.PodDisruptionBudget = &pdb
+				candidate.Spec.DeploymentPlan.Size = 2
+			})
+
+			pdbKey := types.NamespacedName{
+				Name:      brokerCr.Name + "-pdb",
+				Namespace: defaultNamespace,
+			}
+
+			testTimeout := timeout
```

Comment:
is this necessary, we expect the test to succeed once the policy is deployed, so a longer timeout is not a problem.

Maybe we need an init in the test suite and set testTimeout based on USE_EXISTING_CLUSTER, I guess that would sort it in a general sense and the entire suite could revert to testTimeout. but in this case I don't see a problem with using the existingClusterTimeout as we don't wait for that unless there is an error

Reply:
ok, we can just use a longer timeout.